Makefile

```diff
@@ -66,7 +66,7 @@ sanity_check:
 #NOTE:  fails on nonPOSIX-compliant shells (e.g. CMD, powershell)
 #NOTE:  dagster-hex is an external package
 	@echo Checking for prod installs - if any are listed below reinstall with 'pip -e'
-	@! (pip list --exclude-editable | grep -e dagster | grep -v dagster-hex | grep -v dagster-hightouch)
+	@! (pip list --exclude-editable | grep -e dagster | grep -v dagster-hex | grep -v dagster-hightouch | grep -v dagster-insights)
```

Comment:
should you have a pypi install of this instead of an editable install?

Reply:
No-since from internal you can get an editable install.